app/assets/src/components/ui/controls/Checkbox.jsx

```diff
@@ -23,8 +23,9 @@ class Checkbox extends React.Component {
   }
 
   handleClick(event) {
-    const { value, onChange } = this.props;
+    event.stopPropagation();
 
+    const { value, onChange } = this.props;
```

Comment:
Can you kip this at the beginning of the function?

Reply:
👍